<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteEditDialog</class>
 <widget class="QDialog" name="NoteEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NoteEditDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>451</width>
     <height>433</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="leftSideLayout">
    <property name="spacing">
     <number>45</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="titleLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLineEdit" name="titleLineEdit">
        <property name="placeholderText">
         <string>Название заметки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="projectNameLabel">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>в проекте ToDo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="descriptionLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="labelAndEditorLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Описание</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="descriptionTextEdit">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Добавьте подробное описание для Вашей заметки.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonsLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>20</y>
     <width>161</width>
     <height>386</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="rightSideLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <layout class="QGridLayout" name="sidePanelLayout_2">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="joinButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Присоединиться</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="sidePanelLabel_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Предложения</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="sidePanelLayout">
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="membersButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Участники</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="checklistButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Чеклист</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="datesButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Дата</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="attachmentButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Вложения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tagsButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Теги</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="sidePanelLabel">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Добавить к заметке</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
